docker/compose/public_full_node/docker-compose.yaml

```diff
@@ -7,48 +7,48 @@
 # It is intended for use with testnet but can be easily modified for other systems.
 #
 # Testnet genesis and waypoint can be acquired at the following URLs:
-# * https://testnet.libra.org/genesis.blob
-# * https://testnet.libra.org/waypoint.txt
+# * https://testnet.diem.com/genesis.blob
+# * https://testnet.diem.com/waypoint.txt
 # Testnet's genesis does not support onchain discovery and also requires a seed_addrs to be added
 # to the full_node_networks and the discovery method to be set to "none" (this step has been
 # completed):
 #     discovery_method: "none"
 #     seed_addrs:
 #         4223dd0eeb0b0d78720a8990700955e1:
-#             - "/dns4/fn.testnet.libra.org/tcp/6182/ln-noise-ik/b6fd31624af370085cc3f872437bb4d9384b31a11b33b9591ddfaaed5a28b613/ln-handshake/0"
+#             - "/dns4/fn.testnet.diem.com/tcp/6182/ln-noise-ik/b6fd31624af370085cc3f872437bb4d9384b31a11b33b9591ddfaaed5a28b613/ln-handshake/0"
 #
 # TODO:
 #   * Directions on the correct image
 #   * Connecting to the local testnet
 #
 # Additional information:
-# * If you use this compose for different Libra Networks, you will need remove the db volume first.
-# * Libra's testnet produces approximately 3 GB worth of chain data per day, so be patient while
+# * If you use this compose for different Diem Networks, you will need remove the db volume first.
+# * Diem's testnet produces approximately 3 GB worth of chain data per day, so be patient while
 # starting for the first time. As a sanity check, enter the container and check the increasing size
 # of the db:
 #   * `docker exec -it $CONTAINER_ID /bin/bash`
-#   * `du -csm /opt/libra/data``
+#   * `du -csm /opt/diem/data``
 version: "3.8"
 services:
     fullnode:
-        image: libra/validator:testnet
+        image: diem/validator:testnet
```

Comment:
We cannot replace this yet. 

diem on DockerHub is not available. cc @ericnakagawa @JoelMarcey